next.config.js

```diff
@@ -62,12 +62,16 @@ module.exports = withLess({
         decoratorEnv: process.env.DECORATOR_ENV,
         decoratorUrl: process.env.DECORATOR_URL,
         noDecorator: process.env.NO_DECORATOR,
+        veilarboppfolgingClientId: process.env.VEILARBOPPFOLGING_CLIENT_ID,
         veilarboppfolgingHost: process.env.VEILARBOPPFOLGING_HOST,
     },
     publicRuntimeConfig: {
         // Will be available on both server and client
-        loginserviceUrl: process.env.LOGINSERVICE_URL,
-        loginServiceRedirectUrl: process.env.LOGINSERVICE_REDIRECT_URL,
+        idportenWellKnownUrl: process.env.IDPORTEN_WELL_KNOWN_URL,
+        idportenClientId: process.env.IDPORTEN_CLIENT_ID,
+        tokenXClientId: process.env.TOKEN_X_CLIENT_ID,
+        tokenXPrivateJwk: process.env.TOKEN_X_PRIVATE_JWK,
+        tokenXWellKnownUrl: process.env.TOKEN_X_WELL_KNOWN_URL,
```

Comment:
Her er jeg litt på tynn is, men er det riktig å tilgjengeligjøre disse for klienten?